apps/web/public/static/locales/en/common.json

```diff
@@ -2154,6 +2154,8 @@
   "access_bookings": "Read, edit, delete your bookings",
   "allow_client_to_do": "Allow {{clientName}} to do this?",
   "oauth_access_information": "By clicking allow, you allow this app to use your information in accordance with their terms of service and privacy policy. You can remove access in the {{appName}} App Store.",
+  "oauth_form_title":"OAuth client creation form",
+  "oauth_form_description":"This is the form to create a new OAuth client",
```

Comment:
The description pretty much repeats the title so I don't see a reason to keep the description.

Reply:
This is just for future references. Ideally the description and title is same for now but we might have a better description for it in the future right so yeah just for v1